File: namhun/1924.cpp
#include <iostream>

using namespace std;
int main() {
	int m = 0, d = 0;

	cin >> m >> d;

	int c = 0;
	int month[12] = { 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
	for (int i = 1; i < m; i++) {
		c += month[i - 1];
	}
	c += d;

	switch (c % 7) {
	case 0:
		cout << "SUN" << endl;
		break;
	case 1:
		cout << "MON" << endl;
		break;
	case 2:
		cout << "TUE" << endl;
		break;
	case 3:
		cout << "WED" << endl;
		break;
	case 4:
		cout << "THU" << endl;
		break;
	case 5:
		cout << "FRI" << endl;
		break;
	case 6:
		cout << "SAT" << endl;
		break;
	}
	return 0;
}

File: changhyun/1924.java
import java.util.Scanner;
import java.util.StringTokenizer;
public class Main {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in); 
		int A= sc.nextInt();
		int B= sc.nextInt();
		int sum=0;
		
		for (int i = 0; i < A; i++) {
			if(i==1||i==3||i==5||i==7||i==8||i==10||i==12) {
				sum+=31;
			}
			else if(i==4||i==6||i==9||i==11)
				sum+=30;
			else if(i==2)
				sum+=28;
		}
		sum+=B;
		if(sum%7==1)
			System.out.println("MON");
		else if(sum%7==2)
			System.out.println("TUE");
		else if(sum%7==3)
			System.out.println("WED");
		else if(sum%7==4)
			System.out.println("THU");
		else if(sum%7==5)
			System.out.println("FRI");
		else if(sum%7==6)
			System.out.println("SAT");
		else if(sum%7==0)
			System.out.println("SUN");

		sc.close(); 
	}
}
